frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.tsx

```diff
@@ -101,20 +103,28 @@ export const ClassificationForm: React.FC<IClassificationFormProps> = ({
 
   /** determine how to filter the classifications */
   const determineFilter = (val: Number) => {
-    /** non SRES users cannot see subdivided */
-    if (keycloak.hasClaim(Claims.ADMIN_PROPERTIES)) {
-      /** only buildings can be demolished */
-      if (!fieldLabel?.includes('Building')) {
-        return val !== Classifications.Disposed && val !== Classifications.Demolished;
-      } else {
-        return val !== Classifications.Disposed;
-      }
-    } else {
-      return (
-        val !== Classifications.Subdivided &&
-        val !== Classifications.Disposed &&
-        val !== Classifications.Demolished
-      );
+    switch (context) {
```

Comment:
Based on the context display the correct classification options.

Reply:
this does functionally tie this classification form to the inventory sidebar. I'm ok with this given where we are in the project however.

Reply:
ugh, the use of val as a variable name hurts me, not your code though.

Reply:
I'll update to `id` and include a better description for the function.  I was going to add a prop to the component to control, but I couldn't figure out how it is called differently for subdivisions and gave up.  This works for now.